test/testinput/bufr_snow_adpsfc.yaml

```diff
@@ -0,0 +1,113 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./gdas.t06z.adpsfc.tm00.bufr_d"
+
+      mnemonicSets:
+        - mnemonics: [YEAR, MNTH, DAYS, HOUR, MINU, WMOB, WMOS]
+        - mnemonics: [CLAT, CLON]
+        - mnemonics: [SELV, PRES, TMDB, TMDP, TOSD]
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: YEAR
+              month: MNTH
+              day: DAYS
+              hour: HOUR
+              minute: MINU
+
+          longitude:
+            mnemonic: CLON
+
+          latitude:
+            mnemonic: CLAT
+
+          pressureAir:
+            mnemonic: PRES
+
+          stationElevation:
+            mnemonic: SELV
+
+          temperatureAir:
+            mnemonic: TMDB
+
+          temperatureDewpoint:
+            mnemonic: TMDP
+        
+          snowDepth:
+            mnemonic: TOSD
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t06z.adpsfc_snow.tm00.nc"
```

Comment:
I believe this is the cause of the test failure. This path should be:

`./testrun/gdas.t06z.adpsfc_snow.tm00.nc`

The methodology is to put the test output into the ./testrun directory and the expected output in the ./testoutput directory.

Reply:
Steve, the previous branch that I merged was also an adpsfc example, and for that one, I placed the output data into testoutput. I can change the file to be in the testrun directory, but should I also do that for the original adpsfc example? For reference, the file is: ./test/testoutput/gdas.t06z.adpsfc.tm00.nc

Reply:
Just to be clear. The expected result of the test (in the form of a file) goes in the `./testoutput` directory. When the test runs, the output of the bufr converter should be written into the `./testrun` directory into a file with the same name as the expected result file in the `./testoutput` directory.

I think once you fix this path, you are good to go. (It looks like all the other YAML files have their output from the test directed into the `./testrun` directory.)

Reply:
Ah, I see now. I was looking in the wrong place. I just pushed it. Let me know if there's anything else. Thanks